relay-server/src/actors/processor.rs

```diff
@@ -1109,9 +1109,30 @@ impl EnvelopeProcessorService {
                     }
                 }
             }
-            ItemType::ReplayRecording => replays_enabled,
+            ItemType::ReplayRecording => {
+                // XXX: Temporarily, only the Sentry org will be allowed to parse replays while
+                // we measure the impact of this change.
+                if replays_enabled && state.project_state.organization_id == Some(1) {
```

Comment:
I think this is OK as long as it's temporary. Note that this also applies to internal orgs on self-hosted (not sure about single-tenant). In the mid to long term, we should either remove the condition or feature-gate it as outlined in the PR comments.

Reply:
👍